Add offers to account endpoint

alexanderreiff opened the pull request:
As a means to introspect the offers which an account makes, this
implements the account endpoint action for offers.

Fixes #60

Note that the doc-tests here rely on the `trade::All` endpoint being implemented in #127. That will need get merged ahead of this.

### Is there a GIF that reflects how this work made you feel?
![](https://media.giphy.com/media/3ohhwKRT1iDiCsatAA/giphy.gif)

kbacha commented:
FYI, at this point, there will soon be a Limit trait and Order trait that can be derived. Should simplify even more 👍 

It also appears there's a compile err?
https://travis-ci.org/kbacha/stellar-sdk/builds/365400419?utm_source=github_status&utm_medium=notification

alexanderreiff commented:
Updated to use `Limit` and `Order` traits and addressed the silly compile error (made a tiny name change one place but not the order 🤦‍♂️ ).

This still has one failing doc test until #127 merges.
